walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -80,6 +83,7 @@ public void noOptionsForTooLongOfARequest() {
 
   @Test
   public void eventSplitsRestriction() {
+    System.out.println("Start");
```

Comment:
do these print statements help the test?

Reply:
Whoops, these were a artifact of me debugging my tests!